README.md

```diff
@@ -49,6 +49,11 @@ You can also build and run only Ruling Tests using the `ruling` property:
 
     ./gradlew build -Pruling --info --no-daemon
 
+## Validate the published artifact locally
+
+    ./gradlew build -x test publishToMavenLocal
+    find "$HOME/.m2/repository/org/sonarsource/slang/sonar-scala-plugin" -mtime -20s -ls
```

Comment:
The `s` in the `-mtime -20s` argument is not accepted by find 4.8.0 (the default version on Ubuntu).
Since it is also very machine specific, can we remove the time filter altogether?
```suggestion
    find "$HOME/.m2/repository/org/sonarsource/slang/sonar-scala-plugin"  -ls
```